aries_cloudagent/vc/routes.py

```diff
@@ -49,6 +92,16 @@ async def ldp_issue(request: web.BaseRequest):
     context: AdminRequestContext = request["context"]
     body = await request.json()
     credential = VerifiableCredential.deserialize(body["credential"])
+    options = {} if "options" not in body else body["options"]
+    # Default to Ed25519Signature2018 if no proof type was provided
+    if "type" in options:
```

Comment:
While providing a proof type in the options field is not part of the vc-api spec, there is a convention of only using the key `type` instead of `proofType` when doing so. This bit of code will check both and prioritize `proofType` if its present as this is the current acapy field key which is used. It will also default to `Ed25519Signature2018` if nothing is provided. It will the pass whichever value comes out of it as `proofType` to the manager.

Reply:
Might be good to leave this in a comment in the code but otherwise seems reasonable to me